.github/workflows/review-workflow.yml

```diff
@@ -1,6 +1,14 @@
-name: fight-pandemics
+name: review
 
-on: push
+on:
+  push:
+    branches:
+      - feature/**
```

Comment:
@robinv85 this is where we filter for branches on which we want the review build workflow to run. In addition to `bugfix/`, what other prefixes do you think we should filter for?